
#request
string newString

---

#response
string modifiedString
